Views/Products/Detail.cshtml

```diff
@@ -1,27 +1,46 @@
 @model BangazonWeb.ViewModels.ProductDetail
 
 <h2>@Html.DisplayFor(model => model.CurrentProduct.Name)</h2>
+<hr />
 
 <div class="row">
-    <div class="col-md-3">@Html.DisplayFor(model => model.CurrentProduct.Description)</div>
-    <div class="col-md-3">@Html.DisplayFor(model => model.CurrentProduct.Price)</div>
-    <div class="col-md-3">@Html.DisplayFor(model => model.CurrentProduct.DateCreated)</div>
+    <div class="col-md-3">
+        <h5>Description:</h5>
+        <p>@Html.DisplayFor(model => model.CurrentProduct.Description)</p>
+    </div>
+    <div class="col-md-3">
+        <h5>Price:</h5>
+        <p>$@Html.DisplayFor(model => model.CurrentProduct.Price)</p>
+    </div>
+    <div class="col-md-3">
+        <h5>Date Added:</h5>
+        <p>@Html.DisplayFor(model => model.CurrentProduct.DateCreated)</p>
+    </div>
+
+    
     @if (@Model.CurrentProduct.UserId != @Model.ChosenUser.UserId)
     {
+   
         <div class="col-md-3"><a asp-controller="Cart" asp-action="AddToCart" asp-route-id="@Model.CurrentProduct.ProductId">Add to Cart</a></div>
     }
     else
     {
-        <a asp-controller="Products" asp-action="Delete" asp-route-id="@Model.CurrentProduct.ProductId"><span class="glyphicon glyphicon-remove" aria-hidden="true"></span></a>
-        <span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>
-        <a asp-controller="Products" asp-action="Edit" asp-route-id="@Model.CurrentProduct.ProductId"><span class="glyphicon glyphicon-edit" aria-hidden="true"></span></a>
+        <div class="col-md-3">        
+            <a asp-controller="Products" asp-action="Delete" asp-route-id="@Model.CurrentProduct.ProductId"><span class="glyphicon glyphicon-remove" aria-hidden="true"></span></a>

```

Comment:
I think we should make the `Add To Cart` link into a button.  We can just add `class="btn btn-primary btn-xs"` to the `a` tag.